data/users.json

```diff
@@ -314,7 +314,7 @@
     },
     {
       "name": "jonathanhaas",
-      "real_name": "",
+      "real_name": "Jonathan Haas",
```

Comment:
As this file is often refreshed from the Slack API you should also update your profile in the Slack group to have your name so it is not accidentally lost the next time it is updated.